### PR TITLE
Add discord stream alert delete message toggle

### PR DESCRIPTION
**Pull request by robsdedude**
I'm still testing the changes. I don't know of any better way other than to wait until I stream next time and see how the bot behaves.
Feel free to skim over the changes anyways.

### CODE REVIEW

**Review by BrandenB — APPROVED** [review]

**Review by gmt2001 — APPROVED** [review]
Approved, pending your test

File: javascript-source/discord/handlers/streamHandler.js [hunk 1]
```diff
@@ -345,6 +360,15 @@
...
                 $.discord.say(channel, $.discord.userPrefix(mention) + $.lang.get('discord.streamhandler.channel.set', channelName));
             }
+
+            /**
+             * @discordcommandpath streamhandler toggledeletemessage - Toggles if online announcements get deleted after stream.
+             */
+            if (action.equalsIgnoreCase('toggledeletemessage')) {
+                deleteMessageToggle = !deleteMessageToggle;
+                $.inidb.set('discordSettings', 'deleteMessageToggle', deleteMessageToggle);
+                $.discord.say(channel, $.discord.userPrefix(mention) + $.lang.get('discord.streamhandler.delete.toggle', (onlineToggle === true ? $.lang.get('common.enabled') : $.lang.get('common.disabled'))));
```
Comment (BrandenB) [inline javascript-source/discord/handlers/streamHandler.js]: `onlineToggle` should be changed to `deleteMessageToggle `

**Review by BrandenB — CHANGES_REQUESTED** [review]

File: javascript-source/discord/handlers/streamHandler.js (hunk 1 above)
Comment (robsdedude) [reply javascript-source/discord/handlers/streamHandler.js]: Good catch, thanks.

**Comment by gmt2001** [comment]
Just a note: You should never amend/force-push to make new changes. Just make a new commit to the same branch. It helps us track what you've done and when

**Comment by robsdedude** [comment]
Ok, if you prefer to work that way, I'll do better next time.

Do you squash commits when merging? Because in the end it would only obfuscate the history to have these sort of tiny fixes. At least that was my reason to amend and force push (to "pfusch" as I like to call in in German).

**Comment by robsdedude** [comment]
Tests went alright. It does what I expected it to do.

**Review by BrandenB — APPROVED** [review]

**Pull request merged**